src/privateer/cpp/privateer-lib.h

```diff
@@ -98,7 +98,7 @@ namespace privateer
 
         std::string get_colour ( Colour colour, bool original_style, bool inverted = false  );
 
-        inline const std::string get_svg_tooltip ( clipper::MSugar sugar, bool validation )
+        inline const std::string get_svg_tooltip ( clipper::MSugar sugar, bool validation, clipper::MGlycan glycan )
```

Comment:
These things should be passed by reference unless there is a specific reason I don't know about.